master/internal/resourcemanagers/resource_pool.go

```diff
@@ -410,7 +415,15 @@ func (rp *ResourcePool) receiveJobQueueMsg(ctx *actor.Context) error {
 	case job.GetJobQ:
 		ctx.Respond(rp.scheduler.JobQInfo(rp))
 	case job.MoveJob:
-		ctx.Respond(rp.moveJob(msg))
+		response, err := rp.moveJob(msg)
+		ctx.Respond(err)
+		if err == nil {
+			addr, ok := rp.IDToGroupActor[response.JobID]
+			if ok {
+				ctx.Tell(addr, response)
+			}
+		}
```

Comment:
nit https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow
```suggestion
		if err != nil {
			ctx.Respond(err)
			return nil
		}

		addr, ok := rp.IDToGroupActor[response.JobID]
		if !ok {
			return nil
		}
			
		ctx.Tell(addr, response)
```

Reply:
if the ask'er is expecting a response `ctx.Respond(nil)` is implicit.